website/static/js/fangorn.js

```diff
@@ -455,7 +474,7 @@ function doItemOp(operation, to, from, rename, conflict) {
             tb.modal.update(mithrilContent, mithrilButtons);
             return;
         }
-        from.data = resp;
+        from.data = tbOptions.lazyLoadPreprocess.call(this, resp).data;
```

Comment:
Instead of calling `lazyLoadPreprocess` on the options global, why not call it on the instantiated treebeard object instead?